config/logstash.yml

```diff
@@ -281,6 +281,9 @@
 #
 # ------------ Other Settings --------------
 #
+# Run Logstash with root
+run_as.root: true
```

Comment:
This seems backwards to me.  If we're preventing Logstash from running as root, it seems like the default should be `false`.

Reply:
The `config/logstash.yml` we ship should be 100% comments, matching the defaults defined in `environment.rb`.

Reply:
@karenzone let's discuss doc sections in our meeting (on April 27th) and as I mentioned in weekly sync up, initially we offer `NOTICE (non-breaking)` option and let users know that in the future option will be changed (to `BLOCK`).